db-commons/src/main/scala/org/bitcoins/db/DbCommonsColumnMappers.scala

```diff
@@ -1,20 +1,34 @@
 package org.bitcoins.db
 
```

Comment:
A lot of these are kinda hacky, would like review on these if there is a better way to do some of these